src/sentry/notifications/helpers.py

```diff
@@ -472,3 +473,61 @@ def get_most_specific_notification_setting_value(
         )
         or _get_notification_setting_default(ExternalProviders.EMAIL, type)
     )
+
+
+def are_there_any_settings(
+    notification_settings_by_scope: Mapping[
+        NotificationScopeType,
+        Mapping[int, Mapping[ExternalProviders, NotificationSettingOptionValues]],
+    ],
+    recipient: Team | User,
+    parent_id: int,
+    type: NotificationSettingTypes,
+) -> bool:
+    highest_value_by_provider = get_highest_values_by_provider(
+        notification_settings_by_scope,
+        recipient,
+        parent_id,
+        type,
+    )
+    return any(
+        [
+            value != NotificationSettingOptionValues.NEVER
+            for value in highest_value_by_provider.values()
+        ]
+    )
+
+
+def get_highest_values_by_provider(
+    notification_settings_by_scope: Mapping[
+        NotificationScopeType,
+        Mapping[int, Mapping[ExternalProviders, NotificationSettingOptionValues]],
+    ],
+    recipient: Team | User,
+    parent_id: int,
+    type: NotificationSettingTypes,
+) -> Mapping[ExternalProviders, NotificationSettingOptionValues]:
+    highest_value_by_provider = {
+        provider: _get_notification_setting_default(provider, type)
```

Comment:
@mgaeta I'm not quite understanding why the highest value enum takes priority over another enum.